app/backend/src/application/task/change-task-status.ts

```diff
@@ -0,0 +1,55 @@
+
+import StatusModel from 'database/models/status';
+import TaskModel from 'database/models/task';
+import { ChangeTaskStatusPayload } from 'types/transport/requests/task/change-task-status';
+import { ChangeTaskStatusResponsePayload } from 'types/transport/responses/task/change-task-status';
+import Mongoose from 'mongoose';
+
+/**
+ * Updates task status
+ *
+ * @param params - data required to change task status
+ */
+export async function changeTaskStatus(params: ChangeTaskStatusPayload): Promise<ChangeTaskStatusResponsePayload> {
+  const response: ChangeTaskStatusResponsePayload = { };
+  const task = await TaskModel.findById(params.taskId);
+
+  if (!task) {
+    return response;
+  }
+
+  // Remove taskId from previous status tasks array
```

Comment:
comments are not in our code style. Eslint should warn about it.

Reply:
you mean this rule?

https://github.com/codex-team/eslint-config/blob/master/common.json#L183

seems like it's not working for single line comments